th2_common/schema/filter/strategy/impl/default_filter_strategy.py

```diff
@@ -12,7 +12,7 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-
+from fnmatch import fnmatch
```

Comment:
Do we have only one library for wildcards? If no we should test them and take the fastest.

Reply:
I've compared `re` and `fnmatch` modules.
`fnmatch` internally uses `re`, so basically we have only one approach.